HW02_A_Graph-Fails.Rmd

```diff
@@ -46,12 +56,29 @@ ggplot(data = mpg) +
 
 ```
 
+#CORRECTED CODE for Graph 2
+```{r}
+ggplot(mpg,aes(displ,hwy))+
+  geom_point(color="blue")
+```
 
 
 ### Graph Fail 3
 Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
 
+#ANSWER for Graph 3 - 1st mistake
+Alpha is an argument that controls the opacity of the points so setting it to 2 will make the points appear at 20% transparency. To make the points slightly bigger, I would use size instead of alpha. 
+
 Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+#ANSWER for Graph 3 - 2nd mistake
+To change the legend position to the top of the graph, I can simply insert the argument value "top" in the function theme such that the code would read theme(legend.position="top"). 
+
+Another way to do this is to use a numeric vector c(x,y) as an argument in the function theme, especially if I have a very specific location where I would like the legend to appear. However, the values can only be between 0 and 1 in which c(0,0) corresponds to "bottom left" and c(1,1) corresponds to "top right". In the codes for this graph fail, the values was set to be bigger than 1 and instantly, we know that is a mistake. In this case where this person wanted to move the legend to top of the graph such that it is at approximately the point/ordered pair (5,40), he can try code theme(legend.position = c(0.60, 0.80)) and adjust it accordingly from there.

```

Comment:
This is great! Identifying two different methods for generating the same result, there is always multiple ways to do something it seems.

Reply:
I like this method, I'll have to remember it for the future! I spent a lot of time trying to brute-force the coordinate vector system and adjust it to the exact location that I wanted. Simply specifying a function (top, middle, bottom, etc) would've saved so much time!